src/components/base.js

```diff
@@ -1,39 +1,64 @@
-import { PureComponent } from 'react';
 import { injectGlobal } from 'styled-components';
 import reset from 'styled-reset';
 
-export default class extends PureComponent {
-  componentDidMount() {
-    return injectGlobal`
-      ${reset}
-
-      @import url('https://fonts.googleapis.com/css?family=Roboto:300,400,500');
-
-      body, html {
-        font-size: 16px;
-        font-family: "Roboto", Helvetica, sans-serif;
-        -webkit-font-smoothing: antialiased;
-        -moz-osx-font-smoothing: grayscale;
-        line-height: 1.5;
-        height: 100%;
-      }
-
-      * {
-        box-sizing: border-box;
-      }
-
-      *, *:before, *:after {
-        box-sizing: border-box;
-      }
-
-      #___gatsby {
-        height: 100%;
-        display: block;
-      }
-    `;
-  }
-
-  render() {
-    return this.props.children;
-  }
-}
+const baseStyles = () => {
+  return injectGlobal`
+    ${reset}
+
+    @font-face {
+      font-display: fallback;
+      font-family: 'Roboto';
+      font-style: normal;
+      font-weight: 400;
+      src: url(https://fonts.gstatic.com/s/roboto/v18/KFOmCnqEu92Fr1Mu4mxKKTU1Kg.woff2) format('woff2');
+      unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
+    }
+
+    @font-face {
+      font-display: fallback;
+      font-family: 'Roboto';
+      font-style: normal;
+      font-weight: 500;
+      src: url(https://fonts.gstatic.com/s/roboto/v18/KFOlCnqEu92Fr1MmEU9fBBc4AMP6lQ.woff2) format('woff2');
+      unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
+    }
+
+    @font-face {
+      font-display: fallback;
+      font-family: 'Roboto';
+      font-style: normal;
+      font-weight: 900;
+      src: url(https://fonts.gstatic.com/s/roboto/v18/KFOlCnqEu92Fr1MmYUtfBBc4AMP6lQ.woff2) format('woff2');
+      unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
+    }
+
+    body, html {
+      font-size: 16px;
+      font-family: "Roboto", Helvetica, sans-serif;
+      -webkit-font-smoothing: antialiased;
+      -moz-osx-font-smoothing: grayscale;
+      line-height: 1.5;
+      height: 100%;
+    }
+
+    * {
+      box-sizing: border-box;
+    }
+
+    *, *:before, *:after {
+      box-sizing: border-box;
+    }
+
+    #___gatsby {
+      height: 100%;
+      display: block;
+    }
+  `;
+};
+
+const base = props => {
+  baseStyles();
```

Comment:
couldn't this cause the `baseStyles` to be called multiple times?

Reply:
I'm not sure, but this seems to be the recommended way to use `styled-reset`

https://github.com/zacanger/styled-reset